src/groestl-hash-js/groestl.js

```diff
@@ -0,0 +1,202 @@
+/////////////////////////////////////
```

Comment:
Can you add a comment mentioning where you copied the files from?

Reply:
Added.